Lib/test/test_math.py

```diff
@@ -623,6 +624,94 @@ def testFmod(self):
         self.assertEqual(math.fmod(0.0, NINF), 0.0)
         self.assertRaises(ValueError, math.fmod, INF, INF)
 
+    def test_fmax(self):
+        self.assertRaises(TypeError, math.fmax)
+        self.assertRaises(TypeError, math.fmax, 'x', 'y')
+
+        self.assertEqual(math.fmax(0., 0.), 0.)
+        # fmax() does not need to be sensitive to the sign of 0 (F.10.9.2.3).
+        self.assertIn(math.fmax(0., -0.), {-0., 0.})
+        self.assertIn(math.fmax(-0., 0.), {-0., 0.})
```

Comment:
```suggestion
        self.assertEqual(math.fmax(0., -0.), 0.)
        self.assertEqual(math.fmax(-0., 0.), 0.)
```
Equivalence tests ignore sign of zero.